src/components/FeedbackModal.tsx

```
@@ -0,0 +1,277 @@
+          <div className='text-center py-8'>
+            <div className='w-16 h-16 bg-green-100 rounded-full flex items-center justify-center mx-auto mb-4'>
+              <MessageSquare className='h-8 w-8 text-green-600' />
+            </div>
+            <h3 className='text-xl font-semibold text-dark mb-2'>
+              Feedback Received!
+            </h3>
+            <p className='text-gray-600 mb-6'>
+              Thank you for helping us improve Paperlyte. We review all feedback
+              and use it to build a better product.
+            </p>
+            <button onClick={handleClose} className='btn-primary btn-md'>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)